flink-end-to-end-tests/test-scripts/common_s3_minio.sh

```diff
@@ -43,18 +56,18 @@ S3_TEST_DATA_WORDS_URI="s3://$IT_CASE_S3_BUCKET/words"
 #   S3_ENDPOINT
 ###################################
 function s3_start {
-  echo "Spawning minio for s3 tests"
+  echo "Spawning minio for s3 tests with DATA_DIR=$DATA_DIR"
   export MINIO_CONTAINER_ID=$(docker run -d \
-    -P \
-    --mount type=bind,source="$TEST_INFRA_DIR",target=/data \
```

Comment:
@AHeise can you sanity check these changes?

Reply:
LGTM. A few things are quite specific, but given that we will replace these tests with Java soonish, I'd not invest too much time in it.